### ISSUE TITLE
Too many connections error when running large test suite (Laravel4)

### ISSUE THREAD

**Issue by austenc** [Codeception/Codeception#1591]
I have a project that has ~100 functional tests, and I noticed yesterday that when I try to run the entire test suite, some of the latter few tests are erroring with:

`SQLSTATE[08004] [1040] Too many connections`

Note that the failing tests pass just fine if I run only their respective suite(s). For now, I've been able to work around this by putting this in many of my tests:
```
public function _after()
{
    \DB::disconnect('myconnection');
}
```

But it leaves me wondering... is there a better way to handle this? Perhaps some way to set a 'global' `_after` function (so I don't have to repeat this in every test class)? I realize I could make a helper to tear it down and just call that, but there would still be a fair amount of repetition.

Is it possible some optimization could be made in the L4 module? I realize this may be hard to test / reproduce as it's likely just the sheer amount of tests that I have. I'm happy to fiddle around with the codecept source in the context of my project, just let me know where to look! :+1: 

_I'm using codeception 2.0.7 and Laravel 4_

**Comment by mnabialek** [comment]
I've added pull request for that https://github.com/Codeception/Codeception/pull/1665

**Comment by janhenkgerritsen** [comment]
Fixed with #1665.
